repos/system_upgrade/common/actors/cephvolumescan/libraries/cephvolumescan.py

```diff
@@ -0,0 +1,50 @@
+from leapp.libraries.stdlib import run
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM
+import json
+import os
+import re
+
+CEPH_CONF = "/etc/ceph/ceph.conf"
+CONTAINER = "ceph-osd"
+
+def select_osd_container(engine):
+    container_name = ""
+    output = run([engine, 'ps'])
+    for line in output['stdout'].splitlines():
+        container_name = line.split()[-1]
+        if re.match(CONTAINER, container_name):
+            return container_name
+    return container_name
+
+def container_or_rpm():
+    container_binary = ''
+    base_cmd = ['ceph-volume', 'lvm', 'list', '--format', 'json']
+    if has_package(InstalledRPM, "ceph-common"):
+        return base_cmd
+    elif has_package(InstalledRPM, "podman"):
+        container_binary = 'podman'
+    else:
+        container_binary = 'docker'
+    container_name = select_osd_container(container_binary)
+    container_cmd = [container_binary, 'exec', container_name]
+    container_cmd.extend(base_cmd)
+    
+    return container_cmd
+
+def run_ceph_lvm_list():
+    command = container_or_rpm()
+    output = run(command)
+    return json.loads(output['stdout'])
+
+def encrypted_osds_list():
+    if os.path.isfile(CEPH_CONF):
```

Comment:
Thx ! fixed it by putting result = [] out of the if statement it returns an empty list instead of None (which is an acceptable value and does not crash).